test/sickle_test.ts

```diff
@@ -6,22 +6,22 @@ import {expect} from 'chai';
 import {SickleOptions} from '../src/sickle';
 import {annotateSource, transformSource, goldenTests} from './test_support';
 
-let RUN_TESTS_MATCHING: RegExp = null;
+const RUN_TESTS_MATCHING: RegExp = null;
 // RUN_TESTS_MATCHING = /fields/;
 
 // If true, update all the golden .js files to be whatever sickle
-// produces from the .ts source.
-let UPDATE_GOLDENS = true;
+// produces from the .ts source. Do not change this code but run as:
+//     UPDATE_GOLDENS=y gulp test
+const UPDATE_GOLDENS = !!process.env.UPDATE_GOLDENS;
 
 describe('golden tests', () => {
-
   goldenTests().forEach((test) => {
     if (RUN_TESTS_MATCHING && !RUN_TESTS_MATCHING.exec(test.name)) {
       it.skip(test.name);
       return;
     }
     let options: SickleOptions = {};
-    if (/\.untyped\./.test(test.name)) {
+    if (/\.untyped\b/.test(test.name)) {
```

Comment:
It's needed, the new regexp strips the `.in.ts` at the end of the test name. Which is admittedly mostly a side effect, but I find the test names a bit cleaner like that.